spec/code_climate/test_reporter/payload_validator_spec.rb

```diff
@@ -37,6 +37,15 @@ module CodeClimate::TestReporter
       }.to_not raise_error
     end
 
+    it "does not raise if there's a committed_at in ci_service data" do
+      payload[:git][:committed_at] = nil
+      payload[:ci_service] = {}
+      payload[:ci_service][:committed_at] = Time.now
```

Comment:
I think this ought to be `Time.now.to_i.to_s` to reflect the expected format of this value